### PR TITLE
Internal: React History Panel

### PR DESCRIPTION
**Pull request by matipojo**

### CODE REVIEW

File: assets/dev/js/editor/regions/panel/footer.js [hunk 1]
```diff
@@ -134,7 +134,7 @@ module.exports = Marionette.ItemView.extend( {
 	},
 
 	onHistoryClick() {
-		$e.route( 'panel/history/actions' );
+		$e.route( 'panel/history/actions-v2' );
```
Comment (StyleShit) [inline assets/dev/js/editor/regions/panel/footer.js:137]: in the future, we'll probably want to keep the original API (?), so maybe we should keep all those "old" API calls, hook into the route system and replace it to v2 conditionally (based on experiment or whatever)

WDYT?

File: modules/history/assets/js/history/panel-tab.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import ItemView from './item-view';
 import EmptyView from './empty';
 
-module.exports = Marionette.CompositeView.extend( {
+export default Marionette.CompositeView.extend( {
```
Comment (StyleShit) [inline modules/history/assets/js/history/panel-tab.js:4]:
```suggestion
export default class extends CompositeView {
```

File: package.json [hunk 3]
```diff
@@ -8,6 +8,11 @@
     "prev_stable_version": "3.1.4",
     "author": "Elementor Team",
     "homepage": "https://elementor.com/",
+    "workspaces": {
+        "packages": [
+            "packages/editor/packages/*"
+        ]
+    },
```
Comment (StyleShit) [inline package.json:15]: why do we need 2 workspaces? let's just remove the other one...

File: assets/dev/js/editor/regions/panel/footer.js (hunk 1 above)
Comment (matipojo) [reply assets/dev/js/editor/regions/panel/footer.js]: Done

File: modules/history/assets/js/history/panel-tab.js (hunk 2 above)
Comment (matipojo) [reply modules/history/assets/js/history/panel-tab.js:4]: It's not related to the POC

File: package.json (hunk 3 above)
Comment (matipojo) [reply package.json:15]: It's a temp workspace until the package will be published

**Pull request closed without merging**